.github/workflows/build_deploy_rcb_fe.yml

```diff
@@ -0,0 +1,179 @@
+        run: |
+          cat >> ~/.ssh/config <<CONTENT
+          Host rcb
+            HostName $IP_ADDRESS
+            User ${{ secrets.SSH_ADMIN_USERNAME }}
+            IdentityFile ~/.ssh/id_rsa
+          CONTENT
+
+      - name: Install nginx (if not installed)
+        run: |
+          ssh rcb <<CONTENT
+
+          sudo apt -qq update
+          sudo apt -qq install nginx -y
+
+          CONTENT
+
+      - name: Remove Nginx default site (if it exists)
+        continue-on-error: true
+        run: ssh rcb 'sudo rm /etc/nginx/sites-enabled/default'
+        
+      - name: Create config for nginx if it doesn't already exist
+        continue-on-error: true
+        run: |
+          ssh rcb <<'CONTENT'
+
+          sudo mkdir -p /var/www/${{ secrets.DEPLOY_DOMAIN }}/html
+          sudo chown -R ${{ secrets.SSH_ADMIN_USERNAME }}:${{ secrets.SSH_ADMIN_USERNAME }} /var/www/${{ secrets.DEPLOY_DOMAIN }}/html
+          sudo chmod -R 755 /var/www/${{ secrets.DEPLOY_DOMAIN }}
+            
+          sudo tee /etc/nginx/sites-available/${{ secrets.DEPLOY_DOMAIN}} <<'NGINX'
+          server {
+            listen 80 default_server;
+            listen [::]:80 default_server;
+
+            root /var/www/${{ secrets.DEPLOY_DOMAIN }}/html;
+            index index.html;
+
+            server_name ${{ secrets.DEPLOY_DOMAIN }} www.${{secrets.DEPLOY_DOMAIN}};
+
+            location / {
+              try_files $uri $uri/ =404;
+            }
+          }
+          NGINX
+
+          sudo ln -s /etc/nginx/sites-available/${{ secrets.DEPLOY_DOMAIN }} /etc/nginx/sites-enabled/
+
+          CONTENT
+      - name: Download built site
+        uses: actions/download-artifact@v4
+        with:
+          name: production-rcb-fe
+          path: ./rcb-webapp
+
+      - name: Delete old site data (if there is a previous deploy on this instance)
+        run: ssh rcb 'sudo rm -rf /var/www/${{ secrets.DEPLOY_DOMAIN }}/html/*'
+
+      - name: Move built site to server
+        run: scp -r ./rcb-webapp/* rcb:/var/www/${{ secrets.DEPLOY_DOMAIN }}/html
+
+      - name: Restart Nginx
+        run : ssh rcb 'sudo systemctl restart nginx'
+
+      - name: Create / Update DNS Records on domain to point to server (domain)
+        uses: everpcpc/cloudflare-dns-action@v1
+        with:
+          type: "A"
+          name: "${{ secrets.DEPLOY_DOMAIN }}"
+          content: "${{ env.IP_ADDRESS }}"
+          ttl: 1
+          proxied: false
+          token: ${{ secrets.CLOUDFLARE_TOKEN }}
+          zone: ${{ secrets.CLOUDFLARE_ZONE }}
+        
+      - name: Create / Update DNS Records on domain to point to server (www.domain)
+        uses: everpcpc/cloudflare-dns-action@v1
+        with:
+          type: "A"
+          name: "www.${{ secrets.DEPLOY_DOMAIN }}"
+          content: "${{ env.IP_ADDRESS }}"
+          ttl: 1
+          proxied: false
+          token: ${{ secrets.CLOUDFLARE_TOKEN }}
+          zone: ${{ secrets.CLOUDFLARE_ZONE }}
```

Comment:
Aici sunt mai multe records? Daca merge treaba asta atunci ar trebuii sa avem un hostname care duce la aplicatie, in genul:
- en-recipebox
- en-expensesmanager

In plus aici nu are sens sa fie secret deploy domain, nu este sensitive, mai bine il punem la variables